helpers/helpers.ts

```diff
@@ -0,0 +1,9 @@
+export const normalizePrice = (strPrice) => {
```

Comment:
please add type of return value and parameter

Reply:
Done.